src/style.rs

```diff
@@ -231,13 +241,8 @@ impl Style for Entropy {
             }
             entropy *= 1.0f64 / 8.0f64;
 
-            let color = colorgrad::magma().at(entropy);
-            [
-                (color.r * 255.0) as u8,
-                (color.g * 255.0) as u8,
-                (color.b * 255.0) as u8,
-                255,
-            ]
+            let discretized_entropy: usize = ((entropy * 256.0f64) as usize).clamp(0, 256);
```

Comment:
shouldn't that be `.clamp(0, 255)`?

Reply:
Fixed. (I got rid of the magic values and just use the length of the (fixed) array.